src/WebJobs.Script/Description/NodeFunctionInvoker.cs

```diff
@@ -329,6 +323,12 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                 }
             }
 
+            if (input is string)
```

Comment:
Not sure - we convert the types we know about, but there really isn't any catch all conversion. I'll add a TODO in the code to revisit that. Note that we control the static set of input types.